agents/scsi/fence_scsi.py

```diff
@@ -375,6 +386,19 @@ def define_new_opts():
 		"default" : "@VGS_PATH@",
 		"order": 300
 	}
+	all_opt["key_value"] = {
+		"getopt" : ":",
+		"longopt" : "key_value",
+		"help" : "--key_value=<id|hash>          SCSI key node generation method",
+		"required" : "0",
+		"shortdesc" : "Method used to generate the SCSI key. \"id\" (default) \
+uses the positional ID from \"corosync-cmactl nodelist\" output which can get inconsistent \
+when nodes are removed from cluster without full cluster restart. \"hash\" uses part of hash \
+made out of node names which is not affected over time but there is theoretical chance that \
+hashes can collide as size of SCSI key is quite limited.",
```

Comment:
I think we should add a note that it's recommended to use the new hash-method in the longdesc for the agent.

Reply:
What about following? I have tried to make text easy to change in future when default changes from 'id' to 'hash'.
~~~
  having each node in the cluster register a unique key with the SCSI \
- device(s). Once registered, a single node will become the reservation holder \
+ devices(s). Reservation key is generated from \"node id\" (default) or from \
+ \"node name hash\" (recommended) by adjusting \'key-value\' option. \
+ Using hash is recommended to prevent issues when removing nodes \
+ from cluster without full cluster restart. \ 
+ Once registered, a single node will become the reservation holder \
  by creating a \"write exclusive, registrants only\" reservation on the \
~~~
Once the text is confirmed I will commit this together with change to test data that changes with text.

Reply:
commit 58105710876bd6a2220f92ea37d621991d68bf4b